tests/test_browser.py

```diff
@@ -4349,10 +4349,9 @@ def test_fetch_implicit_append(self):
     self.btest('fetch/example_synchronous_fetch.cpp', expected='200', args=['-s', 'FETCH=1', '-s', 'WASM=0', '-s', 'USE_PTHREADS=1', '-s', 'PROXY_TO_PTHREAD=1'])
 
   # Tests synchronous emscripten_fetch() usage from wasm pthread in fastcomp.
-  @no_wasm_backend("fetch API uses an asm.js based web worker to run synchronous XHRs and IDB operations")
   def test_fetch_sync_xhr_in_wasm(self):
```

Comment:
Should this have `@requires_threads`? I forget what that's actually used for.

Reply:
Yes, looks like that's missing from this test (and we don't test on any browser without threads, so it wasn't noticed...). Fixing.